### PR TITLE
Fix Lucas portrait

### PR DESCRIPTION
**Pull request by kosssi**
- Add ">" in Lucas picture
- Add "/" in all pictures
- Add new line for all pictures parameters for respect 80 characters by line

### CODE REVIEW

File: src/about.html [hunk 1]
```diff
@@ -94,7 +94,7 @@ <h1 class="introduction-title">{{__ 'about title'}}</h1>
               </li>
               <li class="ourMembersList-item rounded">
                 <figure class="ourMembersList-item-pic">
-                  <img src="{{ pathTo }}/images/spinner.svg" data-src="{{ pathTo }}/images/about/team/03-lucas.png" class="img-responsive" alt="Portrait of Lucas"
+                  <img src="{{ pathTo }}/images/spinner.svg" data-src="{{ pathTo }}/images/about/team/03-lucas.png" class="img-responsive" alt="Portrait of Lucas">
```
Comment (jsilvestre) [inline src/about.html]: Not trying to be picky here (but being anyway, sorry xD), shouldn't it be "/>"? (actual question)

Comment (m4dz) [inline src/about.html]: So, you want to be coarse @jsilvestre :stuck_out_tongue_closed_eyes:? In fact, no, self-closing tag are necessary in XHTML (as it enforces a strong XML inheritance), but HTML (especially doctype 5) is particularly lazy with this. You can write a lonely tag without slef-closing slash and it' perfectly valid.

More: you can write a tag without the closing tag:
```html
<h1>Hello
<p>it's me:
<ul>
    <li>here
    <li>and there
```
and it's valid too (but not perfectly readable :P)!

**Comment by frankrousseau** [comment]
Can you explain what you do in this PR?

**Comment by kosssi** [comment]
I add PR comment ;)

**Comment by frankrousseau** [comment]
Thx @kosssi

**Pull request merged**